rubyforgood/casa
# Show correct date when editing an existing court date

Steps to test:

1. Make court date that is NOT today, save it
1. Go back and click edit on the court date
1. After this fix is merged, then the editing court date is no longer wrongly today's date
